CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 * **Enhancement**
    * CHANGED: Refactor base costing options parsing to handle more common stuff in a one place [#3125](https://github.com/valhalla/valhalla/pull/3125)
+   * ADDED: Add test for ignore_restrictions behaviour in map-match [#3077](https://github.com/valhalla/valhalla/pull/3077).
```

Comment:
nit: I believe adding entry to the changelog is just recomendation. It definitely makes sense when you fixing some stuff or bringing a new feature. Though I used to omit changelog editing when I pushed small refactoring or other insignificant changes. Probably a new test is an example of such change